sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -325,7 +328,7 @@ export class BatchingReceiver extends MessageReceiver {
           this.name,
           maxWaitTimeInSeconds
         );
-        return finalAction();
+        finalAction();
```

Comment:
finalAction() doesn't need to be returned or await'd?

Reply:
There's no real difference from before. The `actionAfterWaitTimeout` only gets called from a setTimeout. Because of this there isn't anything that's waiting for this function to return. Whether I return the result from `finalAction` or simply invoke it, the result is the same.

The reason a change had to be made at all was because `finalAction` is an async function now, so the return type changed from `void` to `Promise<void>`. I could have kept this as `return finalAction()` and changed the return type of `actionAfterWaitTimeout` to `Promise<void>` to match, but there's no change in behavior either way.

Reply:
Sounds good!